test/unit/power_manager_test.rb

```diff
@@ -73,6 +73,7 @@ class PowerManagerTest < ActiveSupport::TestCase
   def actions_list(host)
     action_map = host.power.send(:action_map)
     actions = PowerManager::SUPPORTED_ACTIONS - action_map.keys.map { |k| k.to_s }
+  
```

Comment:
Robocop doesn't like this line. Don't you have some spaces here? It should be empty line without any trailing white space.

Reply:
yep, there were spaces ... was doing the changes in vi - went over it again in Sublime now.

also tried Rubocop locally - and it found no issues anymore

Reply:
Yep, Jenkins approves and the tests are green now :tada: